www-apps/radicale/radicale-2.1.9.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python{3_4,3_5,3_6} )
+
+inherit distutils-r1 eutils user
+
+MY_PN="Radicale"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="A simple CalDAV calendar server"
+HOMEPAGE="http://radicale.org/"
```

Comment:
The Radicale homepage is available over HTTPS, and even appears to redirect to it. Why add a security hole here?

Reply:
(To be clear, the old ebuild had HTTPS here)

Reply:
done, that was something the old ebuild received inbetween 3183fa37dfdabc8ccfd591ecad68e7a841e4299d